### DISCUSSION TITLE
Ignore notch

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#18395] [Ideas] [answered]
On some phones like the Iphone's or my Mi11 lite 5G there's just a little dot as notch and it kind of annoying to see a small little dot cut out 5% of my screen would be nice if osu would just extend under the notch.

**Comment by smoogipoo** [comment, marked as answer]
The current implementation is a best effort, with improvements planned as per https://github.com/ppy/osu/issues/4330.
